docs/pim/enable_purchasing_products.md

```diff
@@ -118,16 +112,7 @@ ibexa:
                                         not_applicable: true
     ```
 
-You can then assign VAT rates that apply to each product type in the supported regions.
-To do it, in the Back Office, edit the product type, and navigate to the **VAT rates** area.
-
-![Assigning VAT rates to a product type](catalog_vat_rates.png "Assigning VAT rates to a product type")
-
-## Product price
-
-The product must have at least one [price](prices.md) configured.
-
-## Product availability
+Users with sufficient permissions, such as Administrators or Store managers, can then assign VAT rates that apply to every product type in each of the supported regions.
+To do it, in the Back Office, they [open the product type for editing]([[= user_doc =]]/pim/create_product_types/#vat), and navigate to the **VAT rates** area.
```

Comment:
It's developer documentation, user reading this is probably developer and can get sufficient permissions, please go back to active voice addressing the user as 'you' instead of generic 'them'.